apps/web/src/pages/Users/UserInformationPage/components/NotesSection.tsx

```diff
@@ -108,14 +108,20 @@ const NotesSection = ({ user }: BasicUserInformationProps) => {
                   <TextArea
                     id={candidate.id}
                     name={candidate.id}
-                    label={`${intl.formatMessage({
-                      defaultMessage: "Notes",
-                      id: "CSDdh/",
-                      description: "Title for a pool candidates notes field",
-                    })} - ${getFullPoolAdvertisementTitleHtml(
-                      intl,
-                      candidate.pool,
-                    )}`}
+                    label={intl.formatMessage(
+                      {
+                        defaultMessage: "Notes - {poolName}",
+                        id: "Yr4DW5",
+                        description:
+                          "Label for the notes field on the pool candidate application",
```

Comment:
This description should probably be changed if it going to be shared (the id will be changed by doing this as well).

Some clean up of unused strings as well is worthwhile.

## Screenshot

![Screen Shot 2023-03-22 at 14 26 06](https://user-images.githubusercontent.com/3046459/227003071-7faae16a-1d9d-4320-a0b5-23e82c1d01ef.png)

Reply:
okay, done in https://github.com/GCTC-NTGC/gc-digital-talent/pull/6032/commits/1dc617a7604dec107585c2682d275cdbaee03038 and https://github.com/GCTC-NTGC/gc-digital-talent/pull/6032/commits/d9fb5d51c93fa7add57a0ef7001d9ca005b329eb and https://github.com/GCTC-NTGC/gc-digital-talent/pull/6032/commits/65ac4ebdd01ee266fcc520f5f3d560eb49f587d4